MorphicServer/PreferencesEndpoint.cs

```diff
@@ -75,20 +78,16 @@ public async Task Get()
         [Method]
         public async Task Put()
         {
-            var updated = await Request.ReadJson<Preferences>();
-            using (LogContext.PushProperty("RequestedPreferencesUid", Id))
-            {
-                if (updated.Default == null)
-                {
-                    Log.Logger.Warning("Deleting Default preferences. updated.Default is null");
-                }
-                else
-                {
-                    Log.Logger.Information("Updating preferences 'Default'");
-                }
-            }
-            Preferences.Default = updated.Default;
+            var updated = await Request.ReadJson<PreferencesRequest>();
+            Preferences.Default = updated.Default ?? throw new HttpError(HttpStatusCode.BadRequest);
```

Comment:
Shouldn't need the `?? throw` here anymore

Reply:
Nice catch. Thanks.